docs/Splashkit/Modules/Physics/Background/Software_Requirements_Specification.md

```diff
@@ -89,8 +89,22 @@ building a core range of methods. These methods include:
 
 ### 2.1 User Needs
 
+The user requirements for this module are that the module must be usable by game programmers that
+use Splashkit. These requirements include:
+
+- Wrapper for the user's game to be implemented by the user
```

Comment:
Is this true?
Surely only physics objects need to follow prescribed rules rather than the whole game/application.

Like how `apply_gravity(rectangle, time)` could add a downwards force to a rectangle would only require the existence of a rectangle and nothing else (i.e. no game wrapper used here)

Reply:
We could make this "if applicable" in case there are multiple objects in the game (e.g. apply_gravity to an object in the game, but don't let it fall through another fixed object within the game)